aiida/sphinxext/tests/conftest.py

```diff
@@ -7,47 +7,58 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
+"""Pytest fixtures for AiiDA sphinx extension tests."""
 from __future__ import division
 from __future__ import print_function
 from __future__ import absolute_import
 import os
+from os import path
```

Comment:
Why import both `os` and `path` from `os` separately?

Reply:
to make things shorter for `path` commands?
I just fixed a linter complaint here...

Reply:
other functions from os are used.
I think it's fine.